include/roboteam_ai/stp/invariants/NoGoalVisionFromBallInvariant.h

```diff
@@ -0,0 +1,38 @@
+//
+// Created by timovdk on 5/27/20.
+//
+
+#ifndef RTT_NOGOALVISIONFROMBALLINVARIANT_H
+#define RTT_NOGOALVISIONFROMBALLINVARIANT_H
+
+#include <NFParam/Param.h>
+
+#include "BaseInvariant.h"
+
+namespace rtt::ai::stp::invariant {
+/**
+    Calculates goal vision percentage based on the position of the ball
+**/
+class NoGoalVisionFromBallInvariant : public BaseInvariant {
```

Comment:
This is why documentation is useful, because I have the feeling that the documentation does not match the implementation. The class is called NoGoalVision... whereas in the documentation you describe that you compute the percentage of goal vision (and not the percentage of goal that is not visible)...

Reply:
Also make sure to add the date tag when the class is created and the author tag who created this class. Moreover documentation always ends with */ instead of **/.

Reply:
Date and author tag is already there, but thanks